app/models/booking.rb

```diff
@@ -1,4 +1,8 @@
 class Booking < ApplicationRecord
   belongs_to :activity
   belongs_to :user
+
+  validates :start_time, :end_time, :status, :spots_required, presence: true
+  validates :spots_required, numericality: { only_integer: true }
```

Comment:
I did not create a validation for the time to be a time type because there is not other option when they fill the form anyway. But let me know if this is something you would like me to add.

Reply:
Maybe we need to change the schema table data type to 'DateTime' instead of just time, we are not displaying the date of when the activity will occur just the time.